modules/ROOT/pages/notifications/gqlstatus-info-codes.adoc

```diff
@@ -0,0 +1,62 @@
+:description: The page contains GQLSTATUS info codes producing results such as SUCCESS, NO DATA, and OMITTED RESULT.
+
+[[gql-status-info-codes]]
+= Primary GQLSTATUS codes
```

Comment:
As discussed in the weekly meeting, we need to come up with a better title for this page. I believe GQL already uses `primary` for the first GqlStatusObject in the list, which does not need to be one of the ones listed on this page. I don't have a good suggestion on the top of my head unfortunately, but maybe we can start a Slack thread with the rest of the error team and do some brainstorming

Reply:
I can describe it only by negating what it is not. Like `non-notification`. This is also not optimal.